Translated/Dialog/Dachua/trial_apc_7904001.csv

```diff
@@ -0,0 +1,54 @@
+Begin#0,"""/ci1 1 Oh, well hello there. Think you could you lend me a hand here?"""
```

Comment:
/ci1 1 Oh, well hello there. Think you could lend me a hand here?
n.b. line is reused in same file

Reply:
yeah, multiple times. just sega being sega

Reply:
fixed, thank you for pointing this out